examples/Cassie/run_dircon_squatting.cc

```diff
@@ -213,8 +217,19 @@ void GetInitFixedPointGuess(const Vector3d& pelvis_position,
       -1.762, 68.42, 0.652, 1.744;
   mp->SetInitialGuessForAllVariables(init_guess);
 
-  // mp->SetSolverOption(drake::solvers::SnoptSolver::id(),
-  //                     "Print file", "../snopt.out");
+  // Test scaling
```

Comment:
Lets not commit this dead test code to master.

Reply:
Done